config/initializers/simple_form.rb

```diff
@@ -103,7 +103,7 @@
   # config.item_wrapper_class = nil
 
   # How the label text should be generated altogether with the required text.
-  # config.label_text = lambda { |label, required, explicit_label| "#{required} #{label}" }
+  # config.label_text = lambda { |label, required, explicit_label| "#{label}" }
```

Comment:
A question: do you prefer not to display to the user when a field is required?

Reply:
Yes, but I will go with best practice- Done in sha: a2fa585